testing/btest/scripts/base/frameworks/storage/overwriting.zeek

```diff
@@ -7,24 +7,35 @@
 @load base/frameworks/storage/sync
 @load policy/frameworks/storage/backend/sqlite
 
+# Create a typename here that can be passed down into get().
+type str: string;
+
 event zeek_init() {
 	local opts : Storage::BackendOptions;
 	opts$sqlite = [$database_path = "storage-test.sqlite", $table_name = "testing"];
 
-	local key = "key1234";
+	local key = "key1111";
 	local value = "value7890";
 
-	local open_res = Storage::Sync::open_backend(Storage::SQLITE, opts, string, string);
+	local open_res = Storage::Sync::open_backend(Storage::SQLITE, opts, str, str);
 	print "open result", open_res;
 	local b = open_res$value;
 
-	local res = Storage::Sync::put(b, [$key=key, $value=value]);
+	local res = Storage::Sync::put(b, [$key=key, $value=value, $overwrite=F]);
```

Comment:
Probably should use a different value here just in case the overwrite mistakenly happens, line 30 will succeed.